grammar.js

```diff
@@ -1243,8 +1250,9 @@ module.exports = grammar(C, {
     this: _ => 'this',
 
     concatenated_string: $ => seq(
-      choice($.raw_string_literal, $.string_literal),
-      repeat1(choice($.raw_string_literal, $.string_literal)),
+      choice($.identifier, $.string_literal, $.raw_string_literal),
+      choice($.string_literal, $.raw_string_literal),
+      repeat(choice($.identifier, $.string_literal, $.raw_string_literal)),
```

Comment:
Why identifiers in here?

Reply:
Supporting macros, similar to compound_statement in argument_list, minimal state count bump (100) to support a very common use case